contributions/viewLog.ts

```diff
@@ -0,0 +1,69 @@
+// This is a set of helper functions to manage the Epic View Log on the client.
```

Comment:
`rewrap` (VS Code extension) is really helpful for standardising line length for comments like this.